release/models/relay-agent/openconfig-relay-agent.yang

```diff
@@ -624,6 +633,148 @@ module openconfig-relay-agent {
 
   }
 
+  grouping relay-agent-ipv4-helper-address-config {
+    description
+      "Configuration options for an IP helper-address";
+
+    leaf address {
+      type oc-inet:host;
+      description
+        "IP address to relay DHCP requests to";
+    }
+
+    leaf source-address {
+      type union {
+        type inet:ipv4-address;
+        type oc-if:interface-id;
+      }
+      description
+        "IP address or interface whose IP address is to be used as source in
+	relayed DHCP requests";
+    }
+
+    leaf network-instance {
+      type oc-ni:network-instance-ref;
+      description
+        "The network instance that this DHCP Server
+        is associated with";
+    }
+  }
+
+  grouping relay-agent-ipv6-helper-address-config {
+    description
+      "Configuration options for an IPv6 helper-address";
+
+    leaf address {
+      type oc-inet:host;
+      description
+        "IPv6 address to relay DHCPv6 requests to";
+    }
+
+    leaf source-address {
+      type union {
+        type inet:ipv6-address;
```

Comment:
Use OpenConfig types - is there any reason to keep the old IETF types as part of this module import any longer?

```suggestion
        type oc-inet:ipv6-address;
```